cli/src/seth_cli/cli/account_create.go

```diff
@@ -81,7 +81,8 @@ func (args *AccountCreate) Run(config *Config) error {
 	if args.Nonce == "" {
 		nonce, err = client.LookupAccountNonce(priv)
 		if err != nil {
-			return err
+			nonce = 0
```

Comment:
This hides any other error message that would come from `client.LookupAccountNonce`. We need to check the `Account.Nonce` value in `client.LookupAccountNonce` and make sure that a valid value is returned instead of nil

Reply:
there are log messages inside client.LookupAccountNonce for all the error scenarios that i have encountered so far, only when account creation nonce is not generated from rest-api, in other scenarios i do see nonce. this was the only solution i could come up with.